08_voting/index.tex

```diff
@@ -1 +1,101 @@
+
+\textbf{Plurality} will select the relative majority choices as the collective decision. In the IIGO session, the Plurality voting method will be used to decide whether a change of rule goes into effect. When the number of valid ballots (i.e. abstention excluded) is odd, the Plurality method can determine the winner without tied results. Otherwise, the solution to tied results is regimented with the assumption that the proposer of the voting in such events will vote for "agree". Thus, the proposal will be approved if the number of "agree" votes is more than the number of "disagree" votes. 
+
+\textbf{Borda Count} method will allow Voters to rank their preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the winner.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, candidates with the top-most and second-most first-placed votes are chosen among all candidates. If one of them obtains a majority, the Runoff voting can conclude such candidate as the winner. Otherwise, the second round will proceed where the ballots will be recounted by eliminating the candidates other than the top-two. Subsequently, the candidate with the most votes between the top-two candidates will be the winner.
+
+\textbf{Instant Runoff} is a voting method which is used in the elections for a single seat which consist of more than two candidates. 
+Individual voters rank their preference of candidates. The candidate who gets the least number of first-placed is eliminated, and this step is repeated until only one candidate remains.
+
+\textbf{Approval} is a voting method for a single winner in which each voter selects a subset of candidates. Approval will be used for the elections of roles where each voter picks a group of candidates, and the most named or approved candidate wins.
+
+In case of a \textbf{tied election}, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes:
+\begin{itemize}
+    \item The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work when preference lists form a cycle, which is known as the Condorcet paradox or voting paradox.
```

Comment:
suggested re-writing:

 **in terms** of majority relation which ranks the candidates according to one-to-one comparisons.

... the Condorcet winner is the candidate that wins **the majority** in every one-to-one comparison against other candidates. 

is it really "voting paradox" ? I'm asking cuz maybe it has a more complete name :)

Reply:
changed and indeed it is "voting paradox" (at least from Wikipedia - I know that may not sound reliable), had deleted it to prevent confusion though